index.html

```diff
@@ -1,13 +1,33 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title>JS Onboard Project</title>
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	<link rel="stylesheet" href="/Styles.css">
+	<script src="app.js"></script>
```

Comment:
`app.js` makes use of classes in `ApiData.js` and `GridTemp.js`. Please place it after those two.

And then you will have to move the interfaces and functions that are being mentioned by `ApiData.js` and `GridTemp.js` to either one of those two files, or a new file so that they can be available before the code in `ApiData.js` and `GridTemp.js` mentions them.